Mindbox.Expressions/ExpressionExpander.cs

```diff
@@ -210,47 +215,42 @@ protected override Expression VisitUnary(UnaryExpression node)
 			return baseResult;
 		}
 
-
-		private bool TrySubstituteExpression(
+		private Expression SubstituteExpression(
 			Expression expressionExpression, 
 #if NET45 || CORE45 || WINDOWS_PHONE_APP
 			IReadOnlyList<Expression> arguments,
```

Comment:
Wouldn't compile under NET45 || CORE45 || WindowsPhone cause of comma at the end :)
Maybe we can remove all the conditional compilation?